packages/design-system-react/.eslintrc.js

```diff
@@ -16,4 +16,5 @@ module.exports = {
       },
     },
   ],
+  ignorePatterns: ['src/components/icon/icons/*'], // Auto-generated icons
```

Comment:
Adding `src/components/icon/icons/*` to `ignorePatterns` because SVGR's generation process can reorder imports differently than our lint rules expect. Since these are auto-generated files, we don't need to enforce our lint rules on them.